### DISCUSSION TITLE
Any plan to setup CI (ex. Github Action)?

### DISCUSSION THREAD

**Discussion by berviantoleo** [JoshClose/CsvHelper#1642] [Ideas]
Hi, do you have any plan to set up Github Action? I think it will more effective to check the test and the build.

**Comment by JoshClose** [comment]
Yes, I'd like to do that.

**Reply by berviantoleo** [reply]
Thank you.

May I help you to setup? Maybe I will start from some target framework (netstandard & NetCore). After that, I will try for Net Framework.

I try to setup use windows & ubuntu. Seems it can't do at ubuntu/linux?

This is for example https://github.com/berviantoleo/CsvHelper/actions/runs/479503182

Ah, I think someone already tries to fix that: https://github.com/JoshClose/CsvHelper/pull/1627
